frontend/src/pages/SkillsMap/types/index.ts

```diff
@@ -2,7 +2,7 @@ import { BaseSyntheticEvent } from 'react';
 
 import { ICoursesMapResponse, ICourseShortInfo, ICoursesMapElement } from 'types/course';
 
-export interface ICoursesProps {
+export interface ICoursesListProps {
```

Comment:
i suggest to use prop interface direct in component where it is used

Reply:
it should be discussed on the meeting